docs/advanced.rst

```diff
@@ -164,28 +164,28 @@ understand the structure of your URL.
 
     import cherrypy
 
-    @cherrypy.popargs('name')
+    @cherrypy.popargs('band')
     class Band(object):
         def __init__(self):
             self.albums = Album()
 
         @cherrypy.expose
-        def index(self, name):
-            return 'About %s...' % name
+        def index(self, band):
```

Comment:
Here, 'name' is a better word to describe the parameter. It's an attribute of the Band, the "name" of the band.